hooks/pre/30-upgrade.rb

```diff
@@ -59,6 +67,24 @@ def remove_nodes_distributors
   Kafo::Helpers.execute("mongo pulp_database --eval  'db.repo_distributors.remove({'distributor_type_id': \"nodes_http_distributor\"});'")
 end
 
+def fix_pulp_httpd_conf
+  return true unless File.exist?('/etc/httpd/conf.d/pulp.conf.rpmnew')
+
+  Kafo::Helpers.execute('cp /etc/httpd/conf.d/pulp.conf.rpmnew /etc/httpd/conf.d/pulp.conf')
+end
```

Comment:
It is hard to be 100% certain; however, given the types of changes that go in to the z-releases, it seems like this should still be Ok.  That said, we'll need to ensure that we continue to always run --upgrade on those releases and perform proper regression testing.

I did perform multiple --upgrade runs with the installer using these changes and have not seen any adverse affects; however, I do realize that those tests do not involve additional software updates being introduced.